cmd/runner/main.go

```diff
@@ -214,6 +225,18 @@ func main() {
 					}
 				}
 			})
+			if errorOccurred  && !backOff{
+				currentInterval *= factor
+
+				if currentInterval > maxInterval && maxInterval != 0{
+					log.Println("Unable to start workflow after retries", err)
+					os.Exit(1)
```

Comment:
I don't think we should ever exit, just cap the interval so we never sleep longer than 60 seconds.